2025-04-19 17:10:40 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:10:40 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:10:40 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:31:09 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:31:09 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:31:09 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:32:16 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:32:16 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:32:16 [error]: Error: connect ECONNREFUSED 127.0.0.1:587
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16)
2025-04-19 17:57:47 [error]: Error: 000AB758F87F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:355:

2025-04-19 17:57:47 [error]: Error: 000AB758F87F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:355:

2025-04-19 17:57:47 [error]: Error: 000AB758F87F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:355:

2025-04-19 17:58:52 [error]: Error: Invalid login: 534-5.7.9 Application-specific password required. For more information, go to
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor 98e67ed59e1d1-3087e05db51sm2952099a91.43 - gsmtp
    at SMTPConnection._formatError (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:809:19)
    at SMTPConnection._actionAUTHComplete (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:1588:34)
    at SMTPConnection.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:556:26)
    at SMTPConnection._processResponse (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:993:20)
    at SMTPConnection._onData (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:774:14)
    at TLSSocket.SMTPConnection._onSocketData (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:195:44)
    at TLSSocket.emit (node:events:518:28)
    at TLSSocket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-19 17:58:52 [error]: Error: Invalid login: 534-5.7.9 Application-specific password required. For more information, go to
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor 98e67ed59e1d1-3087e05db51sm2952099a91.43 - gsmtp
    at SMTPConnection._formatError (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:809:19)
    at SMTPConnection._actionAUTHComplete (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:1588:34)
    at SMTPConnection.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:556:26)
    at SMTPConnection._processResponse (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:993:20)
    at SMTPConnection._onData (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:774:14)
    at TLSSocket.SMTPConnection._onSocketData (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:195:44)
    at TLSSocket.emit (node:events:518:28)
    at TLSSocket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-19 17:58:52 [error]: Error: Invalid login: 534-5.7.9 Application-specific password required. For more information, go to
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor 98e67ed59e1d1-3087e05db51sm2952099a91.43 - gsmtp
    at SMTPConnection._formatError (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:809:19)
    at SMTPConnection._actionAUTHComplete (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:1588:34)
    at SMTPConnection.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:556:26)
    at SMTPConnection._processResponse (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:993:20)
    at SMTPConnection._onData (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:774:14)
    at TLSSocket.SMTPConnection._onSocketData (/Users/shahil/Desktop/RMS-MERN/backend/node_modules/nodemailer/lib/smtp-connection/index.js:195:44)
    at TLSSocket.emit (node:events:518:28)
    at TLSSocket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-25 13:37:37 [error]: Error: Invalid credentials
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:144:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:5:58)
2025-04-25 13:37:37 [error]: Error: Invalid credentials
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:144:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:5:58)
2025-04-25 13:49:15 [error]: Registration error:
2025-04-25 13:49:15 [error]: Error: Registration session expired. Please start again.
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:87:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-04-25 14:02:38 [error]: Registration error:
2025-04-25 14:02:38 [error]: Error: Invalid or expired OTP
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:81:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS-MERN/backend/src/services/auth.service.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-04-28 13:31:30 [error]: Error: Invalid credentials
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:150:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:5:58)
2025-04-28 13:31:30 [error]: Error: Invalid credentials
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:150:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:5:58)
2025-04-29 19:37:29 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:142:38

  139 }
  140 
  141 // Find user
→ 142 const user = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:37:29 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:142:38

  139 }
  140 
  141 // Find user
→ 142 const user = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:38:01 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:32:46

  29   throw new Error("Invalid email format");
  30 }
  31 
→ 32 const existingUser = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:38:01 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:32:46

  29   throw new Error("Invalid email format");
  30 }
  31 
→ 32 const existingUser = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:39:48 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:142:38

  139 }
  140 
  141 // Find user
→ 142 const user = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:39:48 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:142:38

  139 }
  140 
  141 // Find user
→ 142 const user = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:40:10 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:32:46

  29   throw new Error("Invalid email format");
  30 }
  31 
→ 32 const existingUser = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 19:40:10 [error]: PrismaClientKnownRequestError: 
Invalid `prisma.user.findUnique()` invocation in
/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:32:46

  29   throw new Error("Invalid email format");
  30 }
  31 
→ 32 const existingUser = await prisma.user.findUnique(
The table `public.User` does not exist in the current database.
    at zn.handleRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:7459)
    at zn.handleAndLogRequestError (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6784)
    at zn.request (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:121:6491)
    at async l (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/node_modules/@prisma/client/runtime/library.js:130:9778)
2025-04-29 20:24:19 [error]: Error: Invalid credentials
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:144:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:5:58)
2025-04-29 20:24:19 [error]: Error: Invalid credentials
    at AuthService.<anonymous> (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:144:15)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shahil/Desktop/RMS/pioneer-rms-pos/backend/src/services/auth.service.ts:5:58)
